source/server-discovery-and-monitoring/tests/unified/pool-clear-min-pool-size-error.json

```diff
@@ -0,0 +1,232 @@
+{
+  "description": "pool-cleared-on-min-pool-size-population-error",
+  "schemaVersion": "1.4",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.4",
+      "serverless": "forbid",
+      "topologies": [
+        "single"
+      ]
+    }
+  ],
+  "createEntities": [
+    {
+      "client": {
+        "id": "setupClient",
+        "useMultipleMongoses": false
+      }
+    }
+  ],
+  "tests": [
+    {
+      "description": "Pool is cleared on authentication error during minPoolSize population",
+      "runOnRequirements": [
+        {
+          "minServerVersion": "4.4",
+          "serverless": "forbid",
+          "auth": true
+        }
```

Comment:
Of all the 3 requirements here, only `auth: true` adds something to the top-level `runOnRequirements`. Let's remove the rest, as they achieve nothing.